---
title: "Class 15 Pathway Analysis from RNA-Seq Results"
output: github_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Differential Expression Analysis
```{r}
library(DESeq2)
```

```{r}
# Load our data files
metaFile <- "GSE37704_metadata.csv"
countFile <- "GSE37704_featurecounts.csv"

# Import metadata and take a peak
colData = read.csv(metaFile, row.names=1)
head(colData)
```

Read in count data file
```{r}
# Import countdata
countData = read.csv(countFile, row.names=1)
head(countData)
```

Need to remove the first column in coundData, the length col

```{r}
countData <- countData[,-1]
```

Removing genes with zero counts across all experiments
```{r}
countData <- countData[rowSums(countData)!=0,]
```

How many genes are we left with?

```{r}
nrow(countData)
```

# Principal Component Analysis

The first analysis step is usually always to plot the data but here we have 15k genes. How do we plot this

```{r}
pc <- prcomp( t(countData))
plot(pc)
```

```{r}
summary(pc)
```

```{r}
mycols <- c(rep("blue",3), rep("red",3))
plot(pc$x[,1:2], col=mycols)
```

# DESeq

```{r}
dds = DESeqDataSetFromMatrix(countData=countData,
                             colData=colData,
                             design=~condition)
dds=DESeq(dds)
```

```{r}
dds
```

```{r}
res = results(dds)
res
```

### Volcano plot summary

```{r}
# Make a color vector for all genes
mycols <- rep("gray", nrow(res) )

# Color red the genes with absolute fold change above 2
mycols[ abs(res$log2FoldChange) > 2 ] <- "red"

# Color blue those with adjusted p-value less than 0.01
#  and absolute fold change more than 2
inds <- (res$padj <0.01) & (abs(res$log2FoldChange) > 2 )
mycols[ inds ] <- "blue"

plot(res$log2FoldChange, - log(res$padj), col=mycols, xlab="Log2(FoldChange)", ylab="-Log(P-value)" )

```

# Adding gene annotation

```{r}
library("AnnotationDbi")
library("org.Hs.eg.db")

columns(org.Hs.eg.db)
```


```{r}
res$symbol = mapIds(org.Hs.eg.db,
                    keys=row.names(res), #where are your gene ids
                    keytype="ENSEMBL", #what format are your ids,
                    column="SYMBOL", #what New ids format do you want,
                    multiVals="first")

res$entrez = mapIds(org.Hs.eg.db,
                    keys=row.names(res),
                    keytype="ENSEMBL",
                    column="ENTREZID",
                    multiVals="first")

head(res)
```

#Pathway analysis

Install and load the packages we need

```{r}
##BiocManager::install(c("pathview", "gage","gageData"))
library(pathview)
library(gage)
library(gageData)
data(kegg.sets.hs)
data(sigmet.idx.hs)

# Focus on signaling and metabolic pathways only
kegg.sets.hs = kegg.sets.hs[sigmet.idx.hs]

# Examine the first 3 pathways
head(kegg.sets.hs, 3)
```

The main gage() function requires a named vector of fold changes, where the names of the values are the Entrez gene IDs.

```{r}
foldchanges = res$log2FoldChange
names(foldchanges) = res$entrez
head(foldchanges)
```

Now lets run the gage() function...

```{r}
# Get the results
keggres = gage(foldchanges, gsets=kegg.sets.hs)
```

```{r}
attributes(keggres)
```

Lets look at the first few down regulated pathways

```{r}
head(keggres$less)
```

```{r}
pathview(gene.data=foldchanges, pathway.id="hsa04110")
```
pathview(gene.data=foldchanges, pathway.id="hsa04110")


![My first pathway](./hsa04110.pathview.png)