<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Biological Structure Analysis</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>
<h1> Biological Structure Analysis
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;bio3d&rsquo; version 2.3-4</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
<li><a href="../demo">Code demos</a>.  Use <a href="../../utils/help/demo">demo()</a> to run them.</li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<p style="text-align: center;">
<a href="# "> </a>
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#H">H</a>
<a href="#I">I</a>
<a href="#K">K</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#O">O</a>
<a href="#P">P</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#T">T</a>
<a href="#U">U</a>
<a href="#V">V</a>
<a href="#W">W</a>
<a href="#X">X</a>
<a href="#Z">Z</a>
<a href="#misc">misc</a>
</p>

<table width="100%">
<tr><td style="width: 25%;"><a href="bio3d.package.html">bio3d-package</a></td>
<td>Biological Structure Analysis</td></tr>
</table>

<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="aa.index.html">aa.index</a></td>
<td>AAindex: Amino Acid Index Database</td></tr>
<tr><td style="width: 25%;"><a href="aa.table.html">aa.table</a></td>
<td>Table of Relevant Amino Acids</td></tr>
<tr><td style="width: 25%;"><a href="aa123.html">aa123</a></td>
<td>Convert Between 1-letter and 3-letter Aminoacid Codes</td></tr>
<tr><td style="width: 25%;"><a href="aa2index.html">aa2index</a></td>
<td>Convert an Aminoacid Sequence to AAIndex Values</td></tr>
<tr><td style="width: 25%;"><a href="aa2mass.html">aa2mass</a></td>
<td>Amino Acid Residues to Mass Converter</td></tr>
<tr><td style="width: 25%;"><a href="aa123.html">aa321</a></td>
<td>Convert Between 1-letter and 3-letter Aminoacid Codes</td></tr>
<tr><td style="width: 25%;"><a href="aanma.pdb.html">aanma</a></td>
<td>All Atom Normal Mode Analysis</td></tr>
<tr><td style="width: 25%;"><a href="aanma.pdb.html">aanma.pdb</a></td>
<td>All Atom Normal Mode Analysis</td></tr>
<tr><td style="width: 25%;"><a href="aanma.pdbs.html">aanma.pdbs</a></td>
<td>Ensemble Normal Mode Analysis with All-Atom ENM</td></tr>
<tr><td style="width: 25%;"><a href="aln2html.html">aln2html</a></td>
<td>Create a HTML Page For a Given Alignment</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">amsm.xyz</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="angle.xyz.html">angle.xyz</a></td>
<td>Calculate the Angle Between Three Atoms</td></tr>
<tr><td style="width: 25%;"><a href="example.data.html">annotation</a></td>
<td>Bio3d Example Data</td></tr>
<tr><td style="width: 25%;"><a href="as.fasta.html">as.fasta</a></td>
<td>Alignment to FASTA object</td></tr>
<tr><td style="width: 25%;"><a href="as.pdb.html">as.pdb</a></td>
<td>Convert to PDB format</td></tr>
<tr><td style="width: 25%;"><a href="as.pdb.html">as.pdb.default</a></td>
<td>Convert to PDB format</td></tr>
<tr><td style="width: 25%;"><a href="as.pdb.html">as.pdb.mol2</a></td>
<td>Convert to PDB format</td></tr>
<tr><td style="width: 25%;"><a href="as.pdb.html">as.pdb.prmtop</a></td>
<td>Convert to PDB format</td></tr>
<tr><td style="width: 25%;"><a href="as.select.html">as.select</a></td>
<td>Convert Atomic Indices to a Select Object</td></tr>
<tr><td style="width: 25%;"><a href="is.xyz.html">as.xyz</a></td>
<td>Is an Object of Class 'xyz'?</td></tr>
<tr><td style="width: 25%;"><a href="atom.index.html">atom.index</a></td>
<td>Atom Names/Types</td></tr>
<tr><td style="width: 25%;"><a href="atom.select.html">atom.select</a></td>
<td>Atom Selection from PDB and PRMTOP Structure Objects</td></tr>
<tr><td style="width: 25%;"><a href="atom.select.html">atom.select.mol2</a></td>
<td>Atom Selection from PDB and PRMTOP Structure Objects</td></tr>
<tr><td style="width: 25%;"><a href="atom.select.html">atom.select.pdb</a></td>
<td>Atom Selection from PDB and PRMTOP Structure Objects</td></tr>
<tr><td style="width: 25%;"><a href="atom.select.html">atom.select.prmtop</a></td>
<td>Atom Selection from PDB and PRMTOP Structure Objects</td></tr>
<tr><td style="width: 25%;"><a href="atom2ele.html">atom2ele</a></td>
<td>Atom Names/Types to Atomic Symbols Converter</td></tr>
<tr><td style="width: 25%;"><a href="atom2ele.html">atom2ele.default</a></td>
<td>Atom Names/Types to Atomic Symbols Converter</td></tr>
<tr><td style="width: 25%;"><a href="atom2ele.html">atom2ele.pdb</a></td>
<td>Atom Names/Types to Atomic Symbols Converter</td></tr>
<tr><td style="width: 25%;"><a href="atom2mass.html">atom2mass</a></td>
<td>Atom Names/Types to Mass Converter</td></tr>
<tr><td style="width: 25%;"><a href="atom2mass.html">atom2mass.default</a></td>
<td>Atom Names/Types to Mass Converter</td></tr>
<tr><td style="width: 25%;"><a href="atom2mass.html">atom2mass.pdb</a></td>
<td>Atom Names/Types to Mass Converter</td></tr>
<tr><td style="width: 25%;"><a href="atom2xyz.html">atom2xyz</a></td>
<td>Convert Between Atom and xyz Indices</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="basename.pdb.html">basename.pdb</a></td>
<td>Manipulate PDB File Names</td></tr>
<tr><td style="width: 25%;"><a href="bhattacharyya.html">bhattacharyya</a></td>
<td>Bhattacharyya Coefficient</td></tr>
<tr><td style="width: 25%;"><a href="bhattacharyya.html">bhattacharyya.array</a></td>
<td>Bhattacharyya Coefficient</td></tr>
<tr><td style="width: 25%;"><a href="bhattacharyya.html">bhattacharyya.enma</a></td>
<td>Bhattacharyya Coefficient</td></tr>
<tr><td style="width: 25%;"><a href="bhattacharyya.html">bhattacharyya.matrix</a></td>
<td>Bhattacharyya Coefficient</td></tr>
<tr><td style="width: 25%;"><a href="bhattacharyya.html">bhattacharyya.nma</a></td>
<td>Bhattacharyya Coefficient</td></tr>
<tr><td style="width: 25%;"><a href="bhattacharyya.html">bhattacharyya.pca</a></td>
<td>Bhattacharyya Coefficient</td></tr>
<tr><td style="width: 25%;"><a href="binding.site.html">binding.site</a></td>
<td>Binding Site Residues</td></tr>
<tr><td style="width: 25%;"><a href="bio3d.package.html">bio3d</a></td>
<td>Biological Structure Analysis</td></tr>
<tr><td style="width: 25%;"><a href="biounit.html">biounit</a></td>
<td>Biological Units Construction</td></tr>
<tr><td style="width: 25%;"><a href="blast.pdb.html">blast.pdb</a></td>
<td>NCBI BLAST Sequence Search and Summary Plot of Hit Statistics</td></tr>
<tr><td style="width: 25%;"><a href="bounds.html">bounds</a></td>
<td>Bounds of a Numeric Vector</td></tr>
<tr><td style="width: 25%;"><a href="bounds.sse.html">bounds.sse</a></td>
<td>Obtain A SSE Object From An SSE Sequence Vector</td></tr>
<tr><td style="width: 25%;"><a href="nma.pdb.html">build.hessian</a></td>
<td>Normal Mode Analysis</td></tr>
<tr><td style="width: 25%;"><a href="bwr.colors.html">bwr.colors</a></td>
<td>Color Palettes</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="cat.pdb.html">cat.pdb</a></td>
<td>Concatenate Multiple PDB Objects</td></tr>
<tr><td style="width: 25%;"><a href="chain.pdb.html">chain.pdb</a></td>
<td>Find Possible PDB Chain Breaks</td></tr>
<tr><td style="width: 25%;"><a href="check.utility.html">check.utility</a></td>
<td>Check on Missing Utility Programs</td></tr>
<tr><td style="width: 25%;"><a href="clean.pdb.html">clean.pdb</a></td>
<td>Inspect And Clean Up A PDB Object</td></tr>
<tr><td style="width: 25%;"><a href="cmap.html">cmap</a></td>
<td>Contact Map</td></tr>
<tr><td style="width: 25%;"><a href="cmap.html">cmap.default</a></td>
<td>Contact Map</td></tr>
<tr><td style="width: 25%;"><a href="cmap.html">cmap.pdb</a></td>
<td>Contact Map</td></tr>
<tr><td style="width: 25%;"><a href="cmap.html">cmap.xyz</a></td>
<td>Contact Map</td></tr>
<tr><td style="width: 25%;"><a href="cna.html">cna</a></td>
<td>Protein Dynamic Correlation Network Construction and Community Analysis.</td></tr>
<tr><td style="width: 25%;"><a href="cna.html">cna.dccm</a></td>
<td>Protein Dynamic Correlation Network Construction and Community Analysis.</td></tr>
<tr><td style="width: 25%;"><a href="cna.html">cna.ensmb</a></td>
<td>Protein Dynamic Correlation Network Construction and Community Analysis.</td></tr>
<tr><td style="width: 25%;"><a href="cnapath.html">cnapath</a></td>
<td>Suboptimal Path Analysis for Correlation Networks</td></tr>
<tr><td style="width: 25%;"><a href="com.html">com</a></td>
<td>Center of Mass</td></tr>
<tr><td style="width: 25%;"><a href="com.html">com.pdb</a></td>
<td>Center of Mass</td></tr>
<tr><td style="width: 25%;"><a href="com.html">com.xyz</a></td>
<td>Center of Mass</td></tr>
<tr><td style="width: 25%;"><a href="combine.select.html">combine.select</a></td>
<td>Combine Atom Selections From PDB Structure</td></tr>
<tr><td style="width: 25%;"><a href="community.aln.html">community.aln</a></td>
<td>Align communities from two or more networks</td></tr>
<tr><td style="width: 25%;"><a href="community.tree.html">community.tree</a></td>
<td>Reconstruction of the Girvan-Newman Community Tree for a CNA Class Object.</td></tr>
<tr><td style="width: 25%;"><a href="consensus.html">consensus</a></td>
<td>Sequence Consensus for an Alignment</td></tr>
<tr><td style="width: 25%;"><a href="conserv.html">conserv</a></td>
<td>Score Residue Conservation At Each Position in an Alignment</td></tr>
<tr><td style="width: 25%;"><a href="convert.pdb.html">convert.pdb</a></td>
<td>Renumber and Convert Between Various PDB formats</td></tr>
<tr><td style="width: 25%;"><a href="example.data.html">core</a></td>
<td>Bio3d Example Data</td></tr>
<tr><td style="width: 25%;"><a href="core.find.html">core.find</a></td>
<td>Identification of Invariant Core Positions</td></tr>
<tr><td style="width: 25%;"><a href="core.find.html">core.find.default</a></td>
<td>Identification of Invariant Core Positions</td></tr>
<tr><td style="width: 25%;"><a href="core.find.html">core.find.pdb</a></td>
<td>Identification of Invariant Core Positions</td></tr>
<tr><td style="width: 25%;"><a href="core.find.html">core.find.pdbs</a></td>
<td>Identification of Invariant Core Positions</td></tr>
<tr><td style="width: 25%;"><a href="cov.nma.html">cov.enma</a></td>
<td>Calculate Covariance Matrix from Normal Modes</td></tr>
<tr><td style="width: 25%;"><a href="cov.nma.html">cov.nma</a></td>
<td>Calculate Covariance Matrix from Normal Modes</td></tr>
<tr><td style="width: 25%;"><a href="dccm.xyz.html">cov2dccm</a></td>
<td>DCCM: Dynamical Cross-Correlation Matrix</td></tr>
<tr><td style="width: 25%;"><a href="covsoverlap.html">covsoverlap</a></td>
<td>Covariance Overlap</td></tr>
<tr><td style="width: 25%;"><a href="covsoverlap.html">covsoverlap.enma</a></td>
<td>Covariance Overlap</td></tr>
<tr><td style="width: 25%;"><a href="covsoverlap.html">covsoverlap.nma</a></td>
<td>Covariance Overlap</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="dccm.html">dccm</a></td>
<td>DCCM: Dynamical Cross-Correlation Matrix</td></tr>
<tr><td style="width: 25%;"><a href="dccm.gnm.html">dccm.egnm</a></td>
<td>Dynamic Cross-Correlation from Gaussian Network Model</td></tr>
<tr><td style="width: 25%;"><a href="dccm.enma.html">dccm.enma</a></td>
<td>Cross-Correlation for Ensemble NMA (eNMA)</td></tr>
<tr><td style="width: 25%;"><a href="dccm.gnm.html">dccm.gnm</a></td>
<td>Dynamic Cross-Correlation from Gaussian Network Model</td></tr>
<tr><td style="width: 25%;"><a href="dccm.nma.html">dccm.nma</a></td>
<td>Dynamic Cross-Correlation from Normal Modes Analysis</td></tr>
<tr><td style="width: 25%;"><a href="dccm.pca.html">dccm.pca</a></td>
<td>Dynamic Cross-Correlation from Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="dccm.xyz.html">dccm.xyz</a></td>
<td>DCCM: Dynamical Cross-Correlation Matrix</td></tr>
<tr><td style="width: 25%;"><a href="deformation.nma.html">deformation.nma</a></td>
<td>Deformation Analysis</td></tr>
<tr><td style="width: 25%;"><a href="diag.ind.html">diag.ind</a></td>
<td>Diagonal Indices of a Matrix</td></tr>
<tr><td style="width: 25%;"><a href="difference.vector.html">difference.vector</a></td>
<td>Difference Vector</td></tr>
<tr><td style="width: 25%;"><a href="dist.xyz.html">dist.xyz</a></td>
<td>Calculate the Distances Between the Rows of Two Matrices</td></tr>
<tr><td style="width: 25%;"><a href="dm.html">dm</a></td>
<td>Distance Matrix Analysis</td></tr>
<tr><td style="width: 25%;"><a href="dm.html">dm.pdb</a></td>
<td>Distance Matrix Analysis</td></tr>
<tr><td style="width: 25%;"><a href="dm.html">dm.pdbs</a></td>
<td>Distance Matrix Analysis</td></tr>
<tr><td style="width: 25%;"><a href="dm.html">dm.xyz</a></td>
<td>Distance Matrix Analysis</td></tr>
<tr><td style="width: 25%;"><a href="dssp.html">dssp</a></td>
<td>Secondary Structure Analysis with DSSP or STRIDE</td></tr>
<tr><td style="width: 25%;"><a href="dssp.html">dssp.pdb</a></td>
<td>Secondary Structure Analysis with DSSP or STRIDE</td></tr>
<tr><td style="width: 25%;"><a href="dssp.html">dssp.pdbs</a></td>
<td>Secondary Structure Analysis with DSSP or STRIDE</td></tr>
<tr><td style="width: 25%;"><a href="dssp.html">dssp.xyz</a></td>
<td>Secondary Structure Analysis with DSSP or STRIDE</td></tr>
</table>

<h2><a name="E">-- E --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="elements.html">elements</a></td>
<td>Periodic Table of the Elements</td></tr>
<tr><td style="width: 25%;"><a href="entropy.html">entropy</a></td>
<td>Shannon Entropy Score</td></tr>
<tr><td style="width: 25%;"><a href="example.data.html">example.data</a></td>
<td>Bio3d Example Data</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="load.enmff.html">ff.aaenm</a></td>
<td>ENM Force Field Loader</td></tr>
<tr><td style="width: 25%;"><a href="load.enmff.html">ff.aaenm2</a></td>
<td>ENM Force Field Loader</td></tr>
<tr><td style="width: 25%;"><a href="load.enmff.html">ff.anm</a></td>
<td>ENM Force Field Loader</td></tr>
<tr><td style="width: 25%;"><a href="load.enmff.html">ff.calpha</a></td>
<td>ENM Force Field Loader</td></tr>
<tr><td style="width: 25%;"><a href="load.enmff.html">ff.pfanm</a></td>
<td>ENM Force Field Loader</td></tr>
<tr><td style="width: 25%;"><a href="load.enmff.html">ff.reach</a></td>
<td>ENM Force Field Loader</td></tr>
<tr><td style="width: 25%;"><a href="load.enmff.html">ff.sdenm</a></td>
<td>ENM Force Field Loader</td></tr>
<tr><td style="width: 25%;"><a href="filter.cmap.html">filter.cmap</a></td>
<td>Contact Map Consensus Filtering</td></tr>
<tr><td style="width: 25%;"><a href="filter.dccm.html">filter.dccm</a></td>
<td>Filter for Cross-correlation Matrices (Cij)</td></tr>
<tr><td style="width: 25%;"><a href="filter.identity.html">filter.identity</a></td>
<td>Percent Identity Filter</td></tr>
<tr><td style="width: 25%;"><a href="filter.rmsd.html">filter.rmsd</a></td>
<td>RMSD Filter</td></tr>
<tr><td style="width: 25%;"><a href="fit.xyz.html">fit.xyz</a></td>
<td>Coordinate Superposition</td></tr>
<tr><td style="width: 25%;"><a href="fluct.nma.html">fluct.nma</a></td>
<td>NMA Fluctuations</td></tr>
<tr><td style="width: 25%;"><a href="formula2mass.html">formula2mass</a></td>
<td>Chemical Formula to Mass Converter</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="gap.inspect.html">gap.inspect</a></td>
<td>Alignment Gap Summary</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">geostas</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">geostas.default</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">geostas.enma</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">geostas.nma</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">geostas.pdb</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">geostas.pdbs</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">geostas.xyz</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="blast.pdb.html">get.blast</a></td>
<td>NCBI BLAST Sequence Search and Summary Plot of Hit Statistics</td></tr>
<tr><td style="width: 25%;"><a href="get.pdb.html">get.pdb</a></td>
<td>Download PDB Coordinate Files</td></tr>
<tr><td style="width: 25%;"><a href="get.seq.html">get.seq</a></td>
<td>Download FASTA Sequence Files</td></tr>
<tr><td style="width: 25%;"><a href="gnm.html">gnm</a></td>
<td>Gaussian Network Model</td></tr>
<tr><td style="width: 25%;"><a href="gnm.html">gnm.pdb</a></td>
<td>Gaussian Network Model</td></tr>
<tr><td style="width: 25%;"><a href="gnm.html">gnm.pdbs</a></td>
<td>Gaussian Network Model</td></tr>
</table>

<h2><a name="H">-- H --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="hclustplot.html">hclustplot</a></td>
<td>Dendrogram with Clustering Annotation</td></tr>
<tr><td style="width: 25%;"><a href="example.data.html">hivp</a></td>
<td>Bio3d Example Data</td></tr>
<tr><td style="width: 25%;"><a href="hmmer.html">hmmer</a></td>
<td>HMMER Sequence Search</td></tr>
</table>

<h2><a name="I">-- I --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="identify.cna.html">identify.cna</a></td>
<td>Identify Points in a CNA Protein Structure Network Plot</td></tr>
<tr><td style="width: 25%;"><a href="inner.prod.html">inner.prod</a></td>
<td>Mass-weighted Inner Product</td></tr>
<tr><td style="width: 25%;"><a href="inspect.connectivity.html">inspect.connectivity</a></td>
<td>Check the Connectivity of Protein Structures</td></tr>
<tr><td style="width: 25%;"><a href="is.gap.html">is.gap</a></td>
<td>Gap Characters</td></tr>
<tr><td style="width: 25%;"><a href="is.mol2.html">is.mol2</a></td>
<td>Is an Object of Class 'mol2'?</td></tr>
<tr><td style="width: 25%;"><a href="is.pdb.html">is.pdb</a></td>
<td>Is an Object of Class 'pdb(s)'?</td></tr>
<tr><td style="width: 25%;"><a href="is.pdb.html">is.pdbs</a></td>
<td>Is an Object of Class 'pdb(s)'?</td></tr>
<tr><td style="width: 25%;"><a href="is.select.html">is.select</a></td>
<td>Is an Object of Class 'select'?</td></tr>
<tr><td style="width: 25%;"><a href="is.xyz.html">is.xyz</a></td>
<td>Is an Object of Class 'xyz'?</td></tr>
</table>

<h2><a name="K">-- K --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="example.data.html">kinesin</a></td>
<td>Bio3d Example Data</td></tr>
</table>

<h2><a name="L">-- L --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="layout.cna.html">layout.cna</a></td>
<td>Protein Structure Network Layout</td></tr>
<tr><td style="width: 25%;"><a href="lbio3d.html">lbio3d</a></td>
<td>List all Functions in the bio3d Package</td></tr>
<tr><td style="width: 25%;"><a href="lmi.html">lmi</a></td>
<td>LMI: Linear Mutual Information Matrix</td></tr>
<tr><td style="width: 25%;"><a href="load.enmff.html">load.enmff</a></td>
<td>ENM Force Field Loader</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="mask.dccm.html">mask</a></td>
<td>Mask a Subset of Atoms in a DCCM Object.</td></tr>
<tr><td style="width: 25%;"><a href="mask.dccm.html">mask.dccm</a></td>
<td>Mask a Subset of Atoms in a DCCM Object.</td></tr>
<tr><td style="width: 25%;"><a href="mktrj.html">mktrj</a></td>
<td>PCA / NMA Atomic Displacement Trajectory</td></tr>
<tr><td style="width: 25%;"><a href="mktrj.html">mktrj.enma</a></td>
<td>PCA / NMA Atomic Displacement Trajectory</td></tr>
<tr><td style="width: 25%;"><a href="mktrj.html">mktrj.nma</a></td>
<td>PCA / NMA Atomic Displacement Trajectory</td></tr>
<tr><td style="width: 25%;"><a href="mktrj.html">mktrj.pca</a></td>
<td>PCA / NMA Atomic Displacement Trajectory</td></tr>
<tr><td style="width: 25%;"><a href="bwr.colors.html">mono.colors</a></td>
<td>Color Palettes</td></tr>
<tr><td style="width: 25%;"><a href="motif.find.html">motif.find</a></td>
<td>Find Sequence Motifs.</td></tr>
<tr><td style="width: 25%;"><a href="mustang.html">mustang</a></td>
<td>Structure-based Sequence Alignment with MUSTANG</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="network.amendment.html">network.amendment</a></td>
<td>Amendment of a CNA Network According To A Input Community Membership Vector.</td></tr>
<tr><td style="width: 25%;"><a href="nma.html">nma</a></td>
<td>Normal Mode Analysis</td></tr>
<tr><td style="width: 25%;"><a href="nma.pdb.html">nma.pdb</a></td>
<td>Normal Mode Analysis</td></tr>
<tr><td style="width: 25%;"><a href="nma.pdbs.html">nma.pdbs</a></td>
<td>Ensemble Normal Mode Analysis</td></tr>
<tr><td style="width: 25%;"><a href="normalize.vector.html">normalize.vector</a></td>
<td>Mass-Weighted Normalized Vector</td></tr>
</table>

<h2><a name="O">-- O --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="orient.pdb.html">orient.pdb</a></td>
<td>Orient a PDB Structure</td></tr>
<tr><td style="width: 25%;"><a href="overlap.html">overlap</a></td>
<td>Overlap analysis</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="pairwise.html">pairwise</a></td>
<td>Pair Indices</td></tr>
<tr><td style="width: 25%;"><a href="pca.html">pca</a></td>
<td>Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="pca.array.html">pca.array</a></td>
<td>Principal Component Analysis of an array of matrices</td></tr>
<tr><td style="width: 25%;"><a href="pca.pdbs.html">pca.pdbs</a></td>
<td>Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="pca.tor.html">pca.tor</a></td>
<td>Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="pca.xyz.html">pca.xyz</a></td>
<td>Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="pdb.annotate.html">pdb.annotate</a></td>
<td>Get Customizable Annotations From PDB Or PFAM Databases</td></tr>
<tr><td style="width: 25%;"><a href="pdb.annotate.html">pdb.pfam</a></td>
<td>Get Customizable Annotations From PDB Or PFAM Databases</td></tr>
<tr><td style="width: 25%;"><a href="pdb2aln.html">pdb2aln</a></td>
<td>Align a PDB structure to an existing alignment</td></tr>
<tr><td style="width: 25%;"><a href="pdb2aln.ind.html">pdb2aln.ind</a></td>
<td>Mapping from alignment positions to PDB atomic indices</td></tr>
<tr><td style="width: 25%;"><a href="pdb2sse.html">pdb2sse</a></td>
<td>Obtain An SSE Sequence Vector From A PDB Object</td></tr>
<tr><td style="width: 25%;"><a href="pdbaln.html">pdbaln</a></td>
<td>Sequence Alignment of PDB Files</td></tr>
<tr><td style="width: 25%;"><a href="pdbfit.html">pdbfit</a></td>
<td>PDB File Coordinate Superposition</td></tr>
<tr><td style="width: 25%;"><a href="pdbfit.html">pdbfit.pdb</a></td>
<td>PDB File Coordinate Superposition</td></tr>
<tr><td style="width: 25%;"><a href="pdbfit.html">pdbfit.pdbs</a></td>
<td>PDB File Coordinate Superposition</td></tr>
<tr><td style="width: 25%;"><a href="example.data.html">pdbs</a></td>
<td>Bio3d Example Data</td></tr>
<tr><td style="width: 25%;"><a href="pdbs2pdb.html">pdbs2pdb</a></td>
<td>PDBs to PDB Converter</td></tr>
<tr><td style="width: 25%;"><a href="pdbs2sse.html">pdbs2sse</a></td>
<td>SSE annotation for a PDBs Object</td></tr>
<tr><td style="width: 25%;"><a href="pdbseq.html">pdbseq</a></td>
<td>Extract The Aminoacid Sequence From A PDB Object</td></tr>
<tr><td style="width: 25%;"><a href="pdbsplit.html">pdbsplit</a></td>
<td>Split a PDB File Into Separate Files, One For Each Chain.</td></tr>
<tr><td style="width: 25%;"><a href="pfam.html">pfam</a></td>
<td>Download Pfam FASTA Sequence Alignment</td></tr>
<tr><td style="width: 25%;"><a href="plot.bio3d.html">plot.bio3d</a></td>
<td>Plots with marginal SSE annotation</td></tr>
<tr><td style="width: 25%;"><a href="blast.pdb.html">plot.blast</a></td>
<td>NCBI BLAST Sequence Search and Summary Plot of Hit Statistics</td></tr>
<tr><td style="width: 25%;"><a href="plot.cmap.html">plot.cmap</a></td>
<td>Plot Contact Matrix</td></tr>
<tr><td style="width: 25%;"><a href="plot.cna.html">plot.cna</a></td>
<td>Protein Structure Network Plots in 2D and 3D.</td></tr>
<tr><td style="width: 25%;"><a href="plot.core.html">plot.core</a></td>
<td>Plot Core Fitting Progress</td></tr>
<tr><td style="width: 25%;"><a href="plot.dccm.html">plot.dccm</a></td>
<td>DCCM Plot</td></tr>
<tr><td style="width: 25%;"><a href="plot.dmat.html">plot.dmat</a></td>
<td>Plot Distance Matrix</td></tr>
<tr><td style="width: 25%;"><a href="plot.enma.html">plot.enma</a></td>
<td>Plot eNMA Results</td></tr>
<tr><td style="width: 25%;"><a href="plot.fasta.html">plot.fasta</a></td>
<td>Plot a Multiple Sequence Alignment</td></tr>
<tr><td style="width: 25%;"><a href="plot.fluct.html">plot.fluct</a></td>
<td>Plot Fluctuations</td></tr>
<tr><td style="width: 25%;"><a href="plot.geostas.html">plot.geostas</a></td>
<td>Plot Geostas Results</td></tr>
<tr><td style="width: 25%;"><a href="plot.hmmer.html">plot.hmmer</a></td>
<td>Plot a Summary of HMMER Hit Statistics.</td></tr>
<tr><td style="width: 25%;"><a href="plot.matrix.loadings.html">plot.matrix.loadings</a></td>
<td>Plot Residue-Residue Matrix Loadings</td></tr>
<tr><td style="width: 25%;"><a href="plot.nma.html">plot.nma</a></td>
<td>Plot NMA Results</td></tr>
<tr><td style="width: 25%;"><a href="plot.pca.html">plot.pca</a></td>
<td>Plot PCA Results</td></tr>
<tr><td style="width: 25%;"><a href="plot.pca.loadings.html">plot.pca.loadings</a></td>
<td>Plot Residue Loadings along PC1 to PC3</td></tr>
<tr><td style="width: 25%;"><a href="plot.pca.html">plot.pca.score</a></td>
<td>Plot PCA Results</td></tr>
<tr><td style="width: 25%;"><a href="plot.pca.html">plot.pca.scree</a></td>
<td>Plot PCA Results</td></tr>
<tr><td style="width: 25%;"><a href="plot.rmsip.html">plot.rmsip</a></td>
<td>Plot RMSIP Results</td></tr>
<tr><td style="width: 25%;"><a href="plot.bio3d.html">plotb3</a></td>
<td>Plots with marginal SSE annotation</td></tr>
<tr><td style="width: 25%;"><a href="print.cna.html">print.cna</a></td>
<td>Summarize and Print Features of a cna Network Graph</td></tr>
<tr><td style="width: 25%;"><a href="cnapath.html">print.cnapath</a></td>
<td>Suboptimal Path Analysis for Correlation Networks</td></tr>
<tr><td style="width: 25%;"><a href="print.core.html">print.core</a></td>
<td>Printing Core Positions and Returning Indices</td></tr>
<tr><td style="width: 25%;"><a href="nma.pdbs.html">print.enma</a></td>
<td>Ensemble Normal Mode Analysis</td></tr>
<tr><td style="width: 25%;"><a href="print.fasta.html">print.fasta</a></td>
<td>Printing Sequence Alignments</td></tr>
<tr><td style="width: 25%;"><a href="geostas.html">print.geostas</a></td>
<td>GeoStaS Domain Finder</td></tr>
<tr><td style="width: 25%;"><a href="read.mol2.html">print.mol2</a></td>
<td>Read MOL2 File</td></tr>
<tr><td style="width: 25%;"><a href="nma.pdb.html">print.nma</a></td>
<td>Normal Mode Analysis</td></tr>
<tr><td style="width: 25%;"><a href="pca.xyz.html">print.pca</a></td>
<td>Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="read.pdb.html">print.pdb</a></td>
<td>Read PDB File</td></tr>
<tr><td style="width: 25%;"><a href="read.prmtop.html">print.prmtop</a></td>
<td>Read AMBER Parameter/Topology files</td></tr>
<tr><td style="width: 25%;"><a href="rle2.html">print.rle2</a></td>
<td>Run Length Encoding with Indices</td></tr>
<tr><td style="width: 25%;"><a href="atom.select.html">print.select</a></td>
<td>Atom Selection from PDB and PRMTOP Structure Objects</td></tr>
<tr><td style="width: 25%;"><a href="dssp.html">print.sse</a></td>
<td>Secondary Structure Analysis with DSSP or STRIDE</td></tr>
<tr><td style="width: 25%;"><a href="print.xyz.html">print.xyz</a></td>
<td>Printing XYZ coordinates</td></tr>
<tr><td style="width: 25%;"><a href="project.pca.html">project.pca</a></td>
<td>Project Data onto Principal Components</td></tr>
<tr><td style="width: 25%;"><a href="prune.cna.html">prune.cna</a></td>
<td>Prune A cna Network Object</td></tr>
<tr><td style="width: 25%;"><a href="pymol.html">pymol</a></td>
<td>Biomolecular Visualization with PyMOL</td></tr>
<tr><td style="width: 25%;"><a href="pymol.html">pymol.dccm</a></td>
<td>Biomolecular Visualization with PyMOL</td></tr>
<tr><td style="width: 25%;"><a href="pymol.html">pymol.modes</a></td>
<td>Biomolecular Visualization with PyMOL</td></tr>
<tr><td style="width: 25%;"><a href="pymol.html">pymol.nma</a></td>
<td>Biomolecular Visualization with PyMOL</td></tr>
<tr><td style="width: 25%;"><a href="pymol.html">pymol.pca</a></td>
<td>Biomolecular Visualization with PyMOL</td></tr>
<tr><td style="width: 25%;"><a href="pymol.html">pymol.pdbs</a></td>
<td>Biomolecular Visualization with PyMOL</td></tr>
</table>

<h2><a name="R">-- R --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="read.all.html">read.all</a></td>
<td>Read Aligned Structure Data</td></tr>
<tr><td style="width: 25%;"><a href="read.cif.html">read.cif</a></td>
<td>Read mmCIF File</td></tr>
<tr><td style="width: 25%;"><a href="read.crd.html">read.crd</a></td>
<td>Read Coordinate Data from Amber or Charmm</td></tr>
<tr><td style="width: 25%;"><a href="read.crd.amber.html">read.crd.amber</a></td>
<td>Read AMBER Coordinate files</td></tr>
<tr><td style="width: 25%;"><a href="read.crd.charmm.html">read.crd.charmm</a></td>
<td>Read CRD File</td></tr>
<tr><td style="width: 25%;"><a href="read.dcd.html">read.dcd</a></td>
<td>Read CHARMM/X-PLOR/NAMD Binary DCD files</td></tr>
<tr><td style="width: 25%;"><a href="read.fasta.html">read.fasta</a></td>
<td>Read FASTA formated Sequences</td></tr>
<tr><td style="width: 25%;"><a href="read.fasta.pdb.html">read.fasta.pdb</a></td>
<td>Read Aligned Structure Data</td></tr>
<tr><td style="width: 25%;"><a href="read.mol2.html">read.mol2</a></td>
<td>Read MOL2 File</td></tr>
<tr><td style="width: 25%;"><a href="read.ncdf.html">read.ncdf</a></td>
<td>Read AMBER Binary netCDF files</td></tr>
<tr><td style="width: 25%;"><a href="read.pdb.html">read.pdb</a></td>
<td>Read PDB File</td></tr>
<tr><td style="width: 25%;"><a href="read.pdb.html">read.pdb2</a></td>
<td>Read PDB File</td></tr>
<tr><td style="width: 25%;"><a href="read.pdcBD.html">read.pdcBD</a></td>
<td>Read PQR output from pdcBD File</td></tr>
<tr><td style="width: 25%;"><a href="read.pqr.html">read.pqr</a></td>
<td>Read PQR File</td></tr>
<tr><td style="width: 25%;"><a href="read.prmtop.html">read.prmtop</a></td>
<td>Read AMBER Parameter/Topology files</td></tr>
<tr><td style="width: 25%;"><a href="rgyr.html">rgyr</a></td>
<td>Radius of Gyration</td></tr>
<tr><td style="width: 25%;"><a href="rle2.html">rle2</a></td>
<td>Run Length Encoding with Indices</td></tr>
<tr><td style="width: 25%;"><a href="rmsd.html">rmsd</a></td>
<td>Root Mean Square Deviation</td></tr>
<tr><td style="width: 25%;"><a href="rmsf.html">rmsf</a></td>
<td>Atomic RMS Fluctuations</td></tr>
<tr><td style="width: 25%;"><a href="rmsip.html">rmsip</a></td>
<td>Root Mean Square Inner Product</td></tr>
<tr><td style="width: 25%;"><a href="rmsip.html">rmsip.default</a></td>
<td>Root Mean Square Inner Product</td></tr>
<tr><td style="width: 25%;"><a href="rmsip.html">rmsip.enma</a></td>
<td>Root Mean Square Inner Product</td></tr>
<tr><td style="width: 25%;"><a href="fit.xyz.html">rot.lsq</a></td>
<td>Coordinate Superposition</td></tr>
<tr><td style="width: 25%;"><a href="aanma.pdb.html">rtb</a></td>
<td>All Atom Normal Mode Analysis</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="sdENM.html">sdENM</a></td>
<td>Index for the sdENM ff</td></tr>
<tr><td style="width: 25%;"><a href="seq2aln.html">seq2aln</a></td>
<td>Add a Sequence to an Existing Alignmnet</td></tr>
<tr><td style="width: 25%;"><a href="seqaln.html">seqaln</a></td>
<td>Sequence Alignment with MUSCLE</td></tr>
<tr><td style="width: 25%;"><a href="seqaln.pair.html">seqaln.pair</a></td>
<td>Sequence Alignment of Identical Protein Sequences</td></tr>
<tr><td style="width: 25%;"><a href="seqbind.html">seqbind</a></td>
<td>Combine Sequences by Rows Without Recycling</td></tr>
<tr><td style="width: 25%;"><a href="seqidentity.html">seqidentity</a></td>
<td>Percent Identity</td></tr>
<tr><td style="width: 25%;"><a href="setup.ncore.html">setup.ncore</a></td>
<td>Setup for Running Bio3D Functions using Multiple CPU Cores</td></tr>
<tr><td style="width: 25%;"><a href="sip.html">sip</a></td>
<td>Square Inner Product</td></tr>
<tr><td style="width: 25%;"><a href="sip.html">sip.default</a></td>
<td>Square Inner Product</td></tr>
<tr><td style="width: 25%;"><a href="sip.html">sip.enma</a></td>
<td>Square Inner Product</td></tr>
<tr><td style="width: 25%;"><a href="sip.html">sip.nma</a></td>
<td>Square Inner Product</td></tr>
<tr><td style="width: 25%;"><a href="sse.bridges.html">sse.bridges</a></td>
<td>SSE Backbone Hydrogen Bonding</td></tr>
<tr><td style="width: 25%;"><a href="store.atom.html">store.atom</a></td>
<td>Store all-atom data from a PDB object</td></tr>
<tr><td style="width: 25%;"><a href="dssp.html">stride</a></td>
<td>Secondary Structure Analysis with DSSP or STRIDE</td></tr>
<tr><td style="width: 25%;"><a href="struct.aln.html">struct.aln</a></td>
<td>Structure Alignment Of Two PDB Files</td></tr>
<tr><td style="width: 25%;"><a href="print.cna.html">summary.cna</a></td>
<td>Summarize and Print Features of a cna Network Graph</td></tr>
<tr><td style="width: 25%;"><a href="cnapath.html">summary.cnapath</a></td>
<td>Suboptimal Path Analysis for Correlation Networks</td></tr>
<tr><td style="width: 25%;"><a href="read.pdb.html">summary.pdb</a></td>
<td>Read PDB File</td></tr>
</table>

<h2><a name="T">-- T --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="torsion.pdb.html">torsion.pdb</a></td>
<td>Calculate Mainchain and Sidechain Torsion/Dihedral Angles</td></tr>
<tr><td style="width: 25%;"><a href="torsion.xyz.html">torsion.xyz</a></td>
<td>Calculate Torsion/Dihedral Angles</td></tr>
<tr><td style="width: 25%;"><a href="example.data.html">transducin</a></td>
<td>Bio3d Example Data</td></tr>
<tr><td style="width: 25%;"><a href="trim.pdb.html">trim</a></td>
<td>Trim a PDB Object To A Subset of Atoms.</td></tr>
<tr><td style="width: 25%;"><a href="trim.mol2.html">trim.mol2</a></td>
<td>Trim a MOL2 Object To A Subset of Atoms.</td></tr>
<tr><td style="width: 25%;"><a href="trim.pdb.html">trim.pdb</a></td>
<td>Trim a PDB Object To A Subset of Atoms.</td></tr>
<tr><td style="width: 25%;"><a href="trim.pdbs.html">trim.pdbs</a></td>
<td>Filter or Trim a PDBs Object</td></tr>
<tr><td style="width: 25%;"><a href="trim.xyz.html">trim.xyz</a></td>
<td>Trim a XYZ Object of Cartesian Coordinates.</td></tr>
</table>

<h2><a name="U">-- U --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="unbound.html">unbound</a></td>
<td>Sequence Generation from a Bounds Vector</td></tr>
<tr><td style="width: 25%;"><a href="uniprot.html">uniprot</a></td>
<td>Fetch UniProt Entry Data.</td></tr>
</table>

<h2><a name="V">-- V --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="var.xyz.html">var.pdbs</a></td>
<td>Pairwise Distance Variance in Cartesian Coordinates</td></tr>
<tr><td style="width: 25%;"><a href="var.xyz.html">var.xyz</a></td>
<td>Pairwise Distance Variance in Cartesian Coordinates</td></tr>
<tr><td style="width: 25%;"><a href="vec2resno.html">vec2resno</a></td>
<td>Replicate Per-residue Vector Values</td></tr>
<tr><td style="width: 25%;"><a href="vmd.cna.html">vmd</a></td>
<td>View CNA Protein Structure Network Community Output in VMD</td></tr>
<tr><td style="width: 25%;"><a href="vmd.cna.html">vmd.cna</a></td>
<td>View CNA Protein Structure Network Community Output in VMD</td></tr>
<tr><td style="width: 25%;"><a href="vmd.cna.html">vmd.cnapath</a></td>
<td>View CNA Protein Structure Network Community Output in VMD</td></tr>
<tr><td style="width: 25%;"><a href="vmd_colors.html">vmd_colors</a></td>
<td>VMD Color Palette</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="wrap.tor.html">wrap.tor</a></td>
<td>Wrap Torsion Angle Data</td></tr>
<tr><td style="width: 25%;"><a href="write.crd.html">write.crd</a></td>
<td>Write CRD File</td></tr>
<tr><td style="width: 25%;"><a href="write.fasta.html">write.fasta</a></td>
<td>Write FASTA Formated Sequences</td></tr>
<tr><td style="width: 25%;"><a href="write.mol2.html">write.mol2</a></td>
<td>Write MOL2 Format Coordinate File</td></tr>
<tr><td style="width: 25%;"><a href="write.ncdf.html">write.ncdf</a></td>
<td>Write AMBER Binary netCDF files</td></tr>
<tr><td style="width: 25%;"><a href="write.pdb.html">write.pdb</a></td>
<td>Write PDB Format Coordinate File</td></tr>
<tr><td style="width: 25%;"><a href="write.pir.html">write.pir</a></td>
<td>Write PIR Formated Sequences</td></tr>
<tr><td style="width: 25%;"><a href="write.pqr.html">write.pqr</a></td>
<td>Write PQR Format Coordinate File</td></tr>
</table>

<h2><a name="X">-- X --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="atom2xyz.html">xyz2atom</a></td>
<td>Convert Between Atom and xyz Indices</td></tr>
<tr><td style="width: 25%;"><a href="project.pca.html">xyz2z.pca</a></td>
<td>Project Data onto Principal Components</td></tr>
</table>

<h2><a name="Z">-- Z --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="project.pca.html">z2xyz.pca</a></td>
<td>Project Data onto Principal Components</td></tr>
</table>

<h2><a name="misc">-- misc --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="print.fasta.html">.print.fasta.ali</a></td>
<td>Printing Sequence Alignments</td></tr>
</table>
</body></html>
